Misc/NEWS.d/3.7.0b4.rst

```diff
@@ -46,8 +46,8 @@ Fix potential memory leak in ``normalizestring()``.
 
 Change dict growth function from
 ``round_up_to_power_2(used*2+hashtable_size/2)`` to
-``round_up_to_power_2(used*3)``.  Previously, dict is shrinked only when
-``used == 0``. Now dict has more chance to be shrinked.
+``round_up_to_power_2(used*3)``.  Previously, dict is shrank only when
+``used == 0``. Now dict has more chance to be shrank.
```

Comment:
The original reads better to me (Australian English speaker). If _shrinked_ is not acceptable, I think _shrunk_ is more correct.

Reply:
@vadmium Thank you for your suggestion. I searched the CPython code and found that there are 8 places where 'shrunk' is used, and only here and another file use 'shrinked', so I changed them all to 'shrunk'.